core/keys/function_key.talon-list

```diff
@@ -0,0 +1,26 @@
+list: user.function_key
+-
+f one: f1
```

Comment:
It's a little unfortunate to have to change numbers in multiple places if people customize them. Thoughts?